doc: Documentation review for Readme and AWS docs (#229)
bifravst/bifravst: docs/aws/Authentication.rst

.. _authentication:

Authentication
##############

> The `Cat Tracker Web Application <https://bifravst.github.io/bifravst/docs/app/GettingStarted.html>`_ on AWS connects to the `AWS IoT broker <https://docs.aws.amazon.com/iot/latest/developerguide/iot-connect-devices.html>`_ using WebSockets, and the authentication is done through `AWS Cognito <https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html>`_.

Reviewer: Do not use absolute links here, we will move the docs to https://developer.nordicsemi.com/ in the future (see #56), and this should be doable without changing all links.

This also makes any preview link (e.g. from the downloaded ZIP file) not link to the file in the preview.
```suggestion
The `Cat Tracker Web Application <../app/AWS.html>`_ for AWS connects to the `AWS IoT broker <https://docs.aws.amazon.com/iot/latest/developerguide/iot-connect-devices.html>`_ using WebSockets, and the authentication is done through `AWS Cognito <https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html>`_.
```
